nominees/vrapeutic.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "VRapeutic",
+  "aliases": [
+    ""
+  ],
+  "description": "VRapeutic is a UNICEF Innovation Fund software house specializing in the development of rehabilitation and therapeutic solutions with a focus on virtual reality for learning and neurodevelopmental challenges",
```

Comment:
This description should be rewritten. This should be about the product, not the organization that makes the product

Reply:
@lacabra have contacted VRapeutic on this issue to get a more authentic description, will update once they get back to us.

Reply:
Resolved in commit 328224d